src/python/pants/backend/awslambda/python/awslambda_python_rules.py

```diff
@@ -75,6 +75,13 @@ async def create_python_awslambda(
             entry_point=None,
             output_filename=pex_filename,
             platforms=PexPlatforms([platform]),
+            additional_args=[
+                # Ensure we can resolve manylinux wheels in addition to any AMI-specific wheels.
+                "--manylinux=manylinux2014",
+                # When we're executing Pex on Linux, allow a local interpreter to be resolved if
+                # available and matching the AMI platform.
+                "--resolve-local-platforms",
```

Comment:
We may want to enable this by default for all of our Pex requests, but I want to keep this scoped.